plugins/coverbrowser.koplugin/bookinfomanager.lua

```diff
@@ -621,6 +622,7 @@ function BookInfoManager:extractInBackground(files)
         logger.warn("Failed lauching background extraction sub-process (fork failed)")
         return false -- let caller know it failed
     end
+    UIManager:preventStandby()
```

Comment:
I guess you put it there to not have to put allowStandby above in if not task_pid.
Dunno if the self:terminateBackgroundJobs() above should also be inside preventStandby - it feels like it should, but may be you're fine with small delay, like here just after runInSubProcess is launched.

Reply:
The logic here is: create pid, ban one standby. reap pid, allow one standby. These always come in pairs, nothing else matters. This is necessary because stuck/previous pid killing is ad-hoc with no real top-down flow.

Reply:
OK, it makes sense, I got fooled by the name "allowStandby" :) which doesn't convey the fact that it's some counter that we're incrementing. Might be worth adding a small comment on each of them in this "track pid" cases:
`-- increase/decrease standby preventers count by 1`.

(It feels a bit heavy to track each individual pid, but I guess you have to - even if there's a single place where we're sure all pids are done, the other place where we create pids might get called when there are already old pids created and a standby already prevented. So, ok.)